hacking/aws_config/testing-iam-policy.json.j2

```diff
@@ -0,0 +1,246 @@
+{
```

Comment:
I don't really understand this diff. The file exists in devel, so not sure why it's showing as all new.